README.md

```diff
@@ -1,2 +1,36 @@
 # django-ilmoitin
-Django app for sending notifications
+
+A templated Django messaging library:
+
+# Installation
+
+1. `pip install django-ilmoitin`
+
+2. Add `django-ilmoitin` to `INSTALLED_APPS`.
+
+# Usage
+
+1. Create a `notifications.py` file in django app.
+
+```python
+from django_ilmoitin.registry import notifications
+
+notifications.register("event_created", "Event created") # Register event
+```
+
+2. Import notifications in your apps.py:
+
+```python
+def __init__(self, *args, **kwargs):
+    super(ReservationsConfig, self).__init__(*args, **kwargs)
+    import reservations.notifications
+```
+
```

Comment:
also, please give an example for extending `dummy_context`. I managed to create an in-memory model object and pass it to `dummy_context`, but then it has to be extended in `AppConfig`'s `ready` method.